Trappist/src/Promact.Trappist.Web/wwwroot/app/core/http.service.ts

```diff
@@ -11,7 +11,13 @@ import 'rxjs/add/operator/toPromise';
 
 export class HttpService {
 
-    constructor(private http: Http) {
+    constructor(private http: Http, defaultOptions: RequestOptions) {
+
+        //Prevent request caching for internet explorer
+        defaultOptions.headers.append('Cache-control', 'no-cache');
```

Comment:
everytime it calls an  api the content of the page remained same as it was taking it from the cache , to make the changes get reflected i found this solution